src/transformers/utils/import_utils.py

```diff
@@ -323,7 +324,13 @@ def is_torch_bf16_cpu_available():
 
 
 def is_torch_bf16_available():
-    return is_torch_bf16_cpu_available() or is_torch_bf16_gpu_available()
+    # the original bf16 check was for gpu only, but later a cpu/bf16 combo has emerged so this util
+    # has become ambiguous and therefore deprecated
+    warnings.warn(
+        "The util is_torch_bf16_available is deprecated, please use is_torch_bf16_gpu_available "
+        "or is_torch_bf16_cpu_available instead according to whether it's used with cpu or gpu"
```

Comment:
```suggestion
        "or is_torch_bf16_cpu_available instead according to whether it's used with cpu or gpu",
        FutureWarning
```
We use `FutureWarning` for deprecations usually.

Reply:
Thank you!